qemu/tests/cfg/qemu_disk_img_create.cfg

```diff
@@ -0,0 +1,55 @@
+- qemu_disk_img_create:
+    no qed raw vmdk
+    type = create_external_snapshot_via_protocol
+    kill_vm = yes
+    start_vm = no
+    md5sum_bin = "md5sum"
+    force_create_image = no
+    backup_image_before_testing = yes
+    restore_image_before_testing = yes
+    tmp_dir = /var/tmp
+    image_chain = "image1 sn1"
+    image_name_sn1 = "images/sn1"
+    image_format_sn1 = "qcow2"
+    tmp_file_name = ${tmp_dir}/test.img
+    guest_file_name_sn1 = "${tmp_dir}/sn1"
+    guest_file_name_image1 = "${tmp_dir}/test.img"
+    server_name = localhost
+    file_create_cmd = "dd if=/dev/urandom of=%s bs=4k count=250"
+    Windows:
+        md5sum_bin = "C:\\tools\md5sum.exe"
+        guest_file_name_sn1 = "c:\sn1.img"
+        guest_file_name_image1 = "c:\test.img"
+        x86_64:
+            sync_bin = WIN_UTILS:\Sync\sync64.exe /accepteula
+        i386, i686:
+            sync_bin = WIN_UTILS:\Sync\sync.exe /accepteula
+
+    variants:
+        - protocol_ssh:
+            subcommand = setup_env_for_ssh
+            check_image = no
+            post_check = no
+            image_json_file.driver_sn1 = ssh
+            image_json_file.host = ${server_name}
+            image_json_file.host_key_check = no
+            ssh_key_dir = /root/.ssh
+            create_ssh_key_cmd = "ssh-keygen -t rsa -q -N '' -f /root/.ssh/id_rsa"
+            copy_pub_key_cmd = "cat ${ssh_key_dir}/id_rsa.pub > ${ssh_key_dir}/authorized_keys"
+
+        - protocol_https:
```

Comment:
you write https protocl, but in fact, I not see any place configure cert for https server,  so In fact your httpd use http really not https.